Code/Framework/AzFramework/AzFramework/Quality/QualitySystemBus.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/std/string/string_view.h>
+
+namespace AzFramework
+{
+    class QualitySystemEvents
+        : public AZ::EBusTraits
+    {
+    public:
+        using Bus = AZ::EBus<QualitySystemEvents>;
+
+        // When loading quality group settings using a level value of LevelFromDeviceRules
+        // the device rules will be used to determine the quality level to use.
+        static constexpr int LevelFromDeviceRules = -1;
+
+        // Loads the default quality group level.
+        // @param level Optional quality level.  If LevelFromDeviceRules, the level will be
+        //              determined from existing device rules.
+        virtual void LoadDefaultQualityGroup(int level = LevelFromDeviceRules) = 0;
```

Comment:
I'm a little confused by the use of "Default" here - isn't providing a level value causing it to _not_ use the defaults?

Reply:
Ah yes, "default" refers to the default group, not the default quality level, I'll refine the comment.